php/ubiquity/composer.json

```diff
@@ -0,0 +1,27 @@
+{
+	"require" : {
+		"php" : "^7.1",
+		"phpmv/ubiquity" : "dev-master"
```

Comment:
please stick to the last version, ideally
~~~json
"phpmv/ubiquity" : "~2.3.1"
~~~
to stick to `2.3`

Reply:
I think we should leave the freedom of choice of version to the creators. If I use a dev version, it's to eventually optimize without having to do a PR.
In a permanent run context like on tfb, this is important.